apps/fluent-tester/src/TestComponents/Overflow/OverflowTest.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { ButtonV1 as Button } from '@fluentui-react-native/button';
+import { Divider } from '@fluentui-react-native/divider';
+import { Menu, MenuTrigger, MenuPopover, MenuItem } from '@fluentui-react-native/menu';
+import { Overflow, OverflowItem, useOverflowMenu } from '@fluentui-react-native/overflow';
+import { TextV1 as Text } from '@fluentui-react-native/text';
+
+import MoreHorizontalIcon from './MoreHorizontalFilled.svg';
+import { Test } from '../Test';
+
+const items = ['a', 'b', 'c', 'd', 'e'];
+const itemLabels = {
+  a: 'Item A',
+  b: 'Item B Item B Item B',
+  c: 'Item C Item C',
+  d: 'Item D',
+  e: 'Item E Item E Item E Item E',
+};
+
+const overflowTestPageStyles = StyleSheet.create({
+  containerStyle: {
+    paddingVertical: 4,
+  },
+  width100: {
+    width: 100,
+  },
+  width250: {
+    width: 250,
+  },
+  width400: {
+    width: 400,
+  },
+});
+
+interface OverflowMenuProps {
+  onItemPress: (id: string) => void;
+}
+
+function OverflowMenu(props: OverflowMenuProps) {
+  const { showMenu, menuItemVisibilities, menuRef, onLayout } = useOverflowMenu();
+  if (showMenu) {
+    return (
+      <Menu>
+        <MenuTrigger>
+          <Button
+            accessibilityLabel="More options"
+            onLayout={onLayout}
+            style={{ alignSelf: 'center' }}
```

Comment:
>{ alignSelf: 'center' } [](http://example.com/codeflow?start=19&length=23)

Please model good RN code by memoizing or creating const objects for inputs to props instead of passing in anonymous objects or functions which can cause wasted renders